src/main/java/seedu/address/MainApp.java

```diff
@@ -15,16 +16,18 @@
 import seedu.address.commons.util.StringUtil;
 import seedu.address.logic.Logic;
 import seedu.address.logic.LogicManager;
-import seedu.address.model.AddressBook;
-import seedu.address.model.Model;
-import seedu.address.model.ModelManager;
-import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.Lessons;
 import seedu.address.model.ReadOnlyUserPrefs;
 import seedu.address.model.UserPrefs;
-import seedu.address.model.util.SampleDataUtil;
-import seedu.address.storage.AddressBookStorage;
-import seedu.address.storage.JsonAddressBookStorage;
+import seedu.address.model.modelmanager.management.ManagementModel;
```

Comment:
I recalled that some of you mentioned that it is difficult to conform to the ordering imposed by the style checker. 

I think (high probability) that the ordering is in ascending alphabetical order partitioned by each period, i.e. it is sorted by `package` them `subpackage` then `subsubpackage` (in the form of `package,subpackage.subsubpackage`).

There is a rational behind this ordering as if you want to look through the imports it is easier to skim through via this order. 

Hope this helps!